test/services/apimap-field-builder.test.js

```diff
@@ -12,47 +12,181 @@ const sequelize = new Sequelize(
 
   return sequelize.sync({ force: true })
-    .then(() => new ApimapFieldBuilder(
-      model,
-      _.values(model.rawAttributes)[1],
-      { sequelize: Sequelize },
-    )
-      .perform());
+    .then(() => _.mapValues(model.rawAttributes, (attribute) =>
+      new ApimapFieldBuilder(
+        model,
+        attribute,
+        { sequelize: Sequelize },
+      )
+        .perform()));
 }
 
 describe('services > apimap-field-builder', () => {
   describe('on a UUID column with a UUIDV4 defaultValue', () => {
+    const fieldDefinitions = {
+      uuid: {
+        type: Sequelize.DataTypes.UUID,
+        defaultValue: Sequelize.DataTypes.UUIDV4,
+      },
+    };
+
     it('should have a name uuid', async () => {
       expect.assertions(1);
-      const field = await initializeField();
-      expect(field.field).toStrictEqual('uuid');
+      const { uuid } = await initializeField(fieldDefinitions);
+      expect(uuid.field).toStrictEqual('uuid');
     });
 
     it('should have a String type', async () => {
       expect.assertions(1);
-      const field = await initializeField();
-      expect(field.type).toStrictEqual('String');
+      const { uuid } = await initializeField(fieldDefinitions);
+      expect(uuid.type).toStrictEqual('String');
     });
 
     it('should not be set as required', async () => {
       expect.assertions(1);
-      const field = await initializeField();
-      expect(field.isRequired).toStrictEqual(false);
+      const { uuid } = await initializeField(fieldDefinitions);
+      expect(uuid.isRequired).toStrictEqual(false);
     });
 
     it('should not have a default value', async () => {
       expect.assertions(1);
-      const field = await initializeField();
-      expect(field.defaultValue).toBeUndefined();
+      const { uuid } = await initializeField(fieldDefinitions);
+      expect(uuid.defaultValue).toBeUndefined();
+    });
+  });
+
+  describe('on other default values', () => {
+    it('should handle raw values', async () => {
+      expect.assertions(8);
+
+      const fieldDefinitions = {
+        intValue: {
+          type: Sequelize.DataTypes.INTEGER,
+          defaultValue: 42,
+        },
+        stringValue: {
+          type: Sequelize.DataTypes.STRING,
+          defaultValue: 'default_value',
+        },
+      };
+      const { intValue, stringValue } = await initializeField(fieldDefinitions);
+
+      expect(intValue.field).toStrictEqual('intValue');
+      expect(intValue.type).toStrictEqual('Number');
+      expect(intValue.isRequired).toBeUndefined();
+      expect(intValue.defaultValue).toStrictEqual(42);
+
+      expect(stringValue.field).toStrictEqual('stringValue');
+      expect(stringValue.type).toStrictEqual('String');
+      expect(stringValue.isRequired).toBeUndefined();
+      expect(stringValue.defaultValue).toStrictEqual('default_value');
+    });
+  });
+
+  describe('on Sequelize.Utils.Literal values', () => {
+    it('should handle simple values', async () => {
```

Comment:
Same here, `should handle simple values` is a bit vague

Reply:
Ditto.